src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,50 @@
+.nav {
+
+  &__list {
+    display: flex;
+  }
+
+  &__item {
+    position: relative;
+    margin-left: 20px;
+    display: flex;
+
+    &:first-child {
+      margin-left: 0;
+    }
+  }
+
+  &__link {
+    padding: 23px 0;
+
+    font-size: 12px;
+    font-weight: $font-w-s;
+    line-height: 14px;
+    text-transform: uppercase;
+    color: $card-text-color;
+    white-space: nowrap;
+
+    transition: color 0.3s;
```

Comment:
Would be nice to create a variable for duration, then we can use it for every transition to make it's duration consistent with other page transitions